### PR TITLE
[WIP] elasticsearch 5 with/without searchguard

### PR DESCRIPTION
**Pull request by t0ffel**
Initial images for Elasticsearch 5 with and without searchguard5.

### CODE REVIEW

File: elasticsearch/Dockerfile [hunk 1]
```diff
@@ -30,11 +29,10 @@ ADD elasticsearch.repo /etc/yum.repos.d/elasticsearch.repo
 RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
     yum install -y --setopt=tsflags=nodocs \
                 java-1.8.0-openjdk-headless \
-                elasticsearch-${ES_VER} \
                 PyYAML && \
+    yum update -y && \
```
Comment (richm) [inline elasticsearch/Dockerfile]: Do we need to backport this?

File: elasticsearch/install.sh [hunk 2]
```diff
@@ -3,23 +3,21 @@
 set -ex
 
 mkdir -p ${HOME}
+cd ${HOME}/
+curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VER}.rpm
+rpm -ivh elasticsearch-${ES_VER}.rpm
```
Comment (richm) [inline elasticsearch/install.sh]: yum install doesn't work?
What about
```
yum -y install https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${ES_VER}.rpm
```
?

Comment (t0ffel) [inline elasticsearch/install.sh]: fixed

File: elasticsearch/Dockerfile (hunk 1 above)
Comment (t0ffel) [inline elasticsearch/Dockerfile]: probably

**Comment by t0ffel** [comment]
ptal @richm @jcantrill @portante

**Comment by portante** [comment]
@t0ffel, can you rebase your changes?  They are way out of date it seems (90 commits ...).

**Comment by t0ffel** [comment]
@portante they are rebased against master, it's just that I'm committing to the 5.x branch which is out of date

**Comment by portante** [comment]
Can we get that branch up to date with master first so that your changes are not kept alongside all the rest?

**Comment by lukas-vlcek** [comment]
@t0ffel After reading `man curl` more carefully shouldn't we just __add__ `--head` to all `curl -X HEAD` instead of __replacing__ it? So in the end we should end up with `curl -X HEAD --head`. See #655

**Comment by jcantrill** [comment]
closing in favor of #892

**Pull request closed without merging**